poleno/utils/tests/test_misc.py

```diff
@@ -224,12 +224,11 @@ def test_application_octet_stream(self):
     def test_application_pdf(self):
         self.assertEqual(guess_extension(u'application/pdf'), u'.pdf')
 
-    @unittest.skip(u'FIXME')
     def test_unknown_content_type(self):
-        self.assertIsNone(guess_extension(u'application/nonexistent'))
+        self.assertEqual(guess_extension(u'application/nonexistent'), u'.bin')
```

Comment:
Bude este treba spravit test pre `None`, ale to az v samostatnom PR pre doplnanie testov.

Reply:
@viliambalaz Asi ho rovno pridaj teraz, nech nezabudneme nan.

Reply:
Prehodnotil som pripad s `None`, asi nebude ten test potrebny, lebo je v postate pokryty s `test_unknown_content_type_with_default_extension`. Tak spravim podla tvojho uvazenia ci ho tam pridat alebo nie.

Reply:
@viliambalaz Tak ako si spravil test `test_unknown_content_type_with_default_extension_None` je dobre. Akurat, ze by som ten test nazval `test_unknown_content_type_with_no_default_extension`.

Reply:
:ok: